source/OptimaJet.DWKit.StarterApplication/Repositories/OrganizationRepository.cs

```diff
@@ -18,11 +18,12 @@ public class OrganizationRepository : BaseRepository<Organization>
             ILoggerFactory loggerFactory,
             IJsonApiContext jsonApiContext,
             CurrentUserRepository currentUserRepository,
+            StatusUpdateService statusUpdateService,
             IDbContextResolver contextResolver
-            ) : base(loggerFactory, jsonApiContext, currentUserRepository, contextResolver)
+            ) : base(loggerFactory, jsonApiContext, currentUserRepository, statusUpdateService, contextResolver)
         {
         }
-        public override IQueryable<Organization> Filter(IQueryable<Organization> query, FilterQuery filterQuery)
+        public override IQueryable<Organization> Filter(IQueryable<Organization> entities, FilterQuery filterQuery)
```

Comment:
why all these types of changes?

Reply:
There were warnings in visual studio that the name didn't match the original.